tests/conftest.py

```
@@ -1441,3 +1497,35 @@ def clean_migration_home(
     state_file.unlink(missing_ok=True)
 
     yield home
+
+
+@pytest.fixture(scope="session", autouse=True)
+def _install_sqlite_provenance():
```

Ruff (0.15.10)

[warning] Missing return type annotation for private function `_install_sqlite_provenance`

(ANN202)